nservicebus/azure-service-bus/error-handling.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Azure Service Bus Error Handling
+reviewed: 2016-08-29
+tags:
+- Azure
+- Cloud
+- Error Handling
+---
+
+## Native Error Handling
+
+Next to the [error handling capabilities offered by NServiceBus](/nservicebus/recoverability/configure-error-handling.md), the Azure Service Bus also provides error handling capabilities of its own. This document describes the native error handling capabilities and compensating actions should there be a need. 
+
+### Azure Service Bus SDK
+
+In a cloud environment, exceptions are not exceptional at all. Given the size and complexity of a cloud environment, connectivity and service side capacity problems are to be expected. For a large part, these problems are covered by leveraging the retry behavior on transient exceptions of the Azure Service Bus SDK, known as [`RetryPolicy`](https://azure.microsoft.com/en-us/documentation/articles/best-practices-retry-service-specific/#service-bus-retry-guidelines).
+
+The following document describes a list of [common exceptions](https://azure.microsoft.com/en-us/documentation/articles/service-bus-messaging-exceptions/) that may be logged by NServiceBus. Usually, these exceptions are transient in their nature and will be automatically resolved by the retry policy of the Azure Service Bus SDK, or they are resolved at the transport level by reestablishing the connection to the service.
```

Comment:
Indeed, we log and keep going, as these exceptions are usually transient they will go away after a few minutes. But we still log them so that user can use the tracking id to contact support if the problem persists.